packages/server/src/file-ingestion/file-ingestion.service.ts

```
@@ -0,0 +1,228 @@
+        vector: chunk.metadata.embedding as number[],
+        content: chunk.text,
+        metadata: {
+          documentId: chunk.metadata.documentId,
+          chunkIndex: chunk.metadata.chunkIndex,
+          originalName: meta.originalName,
+          mimeType: meta.mimeType,
+        },
+      }));
+      await this.vectorStore.upsert(agentId, upsertPayload);
+    } catch (err) {
+      this.logger.error('Embedding failed', err as any);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)